src/main/java/net/openhft/chronicle/wire/channel/ChronicleChannelCfg.java

```diff
@@ -74,9 +74,9 @@ public Set<HostPortCfg> hostPorts() {
      * @param port     the port number
      * @return this configuration instance
      */
-    public ChronicleChannelCfg<C> addHostnamePort(String hostname, int port) {
+    public C addHostnamePort(String hostname, int port) {
```

Comment:
I wonder if this signature change will break any callsites, if so we may need to put this signature through a deprecation cycle before changing it

Reply:
It's only possible to do that by adding a new method. This functionality is not widely used.

Reply:
It does not seem right. Why not simply return raw ChronicleChannelCfg? 
I don't see any parametrization used in the method.

Reply:
It's needed so that subclasses have the right co-variant return type.